docs/en/start/configuration.md

```diff
@@ -110,14 +167,14 @@ final class HttpConfig extends InjectableConfig
 ```
 
 > **Note**
-> See the reference for each component configuration in the related documentation section. 
+> See the reference for each component configuration in the related documentation section.
 
 ## Accessing Configuration Values
 
 ### Config objects
 
-All of the config objects in the Spiral Framework are injectable. It means that when you try to resolve a config object 
-via container, it will automatically load all values from config file or will use default settings.
+In the Spiral Framework, all config objects are injectable.
+This means that when you try to resolve a config object through the container, it will automatically load all values from the config file or use default settings.
```

Comment:
```suggestion
This means that when you try to resolve a config object through the container, it will automatically load all values from the config file or will use default settings.
```